src/api/helpers/map-feedback-satisfaction/index.ts

```diff
@@ -0,0 +1,12 @@
+import { FEEDBACK } from '../../constants';
+
+/**
+ * mapFeedbackSatisfaction
+ * maps satisfaction value from user selection
+ * returns formatted text value for email
+ * @param {String} satisfaction
+ * @returns {String} mapped satisfaction email text
+ */
+const mapFeedbackSatisfaction = (satisfaction: string) => FEEDBACK.EMAIL_TEXT[satisfaction];
```

Comment:
If a satisfaction is not found, will things break?

Reply:
it should never not be found as the schema won't allow it to be submitted to the api.  And if null, then this won't be called